lib/main.coffee

```diff
@@ -7,47 +7,68 @@ module.exports =
       description: 'Lint files while typing, without the need to save them'
       type: 'boolean'
       default: true
+      order: 1
+
+    ignoredMessageTypes:
+      title: "Ignored message Types"
+      type: 'array'
+      default: []
+      items:
+        type: 'string'
+      order: 2
+
+    showErrorInline:
+      title: 'Show Inline Tooltips'
+      description: 'Show inline tooltips for errors'
+      type: 'boolean'
+      default: true
+      order: 3
+    underlineIssues:
+      title: 'Underline Issues'
+      type: 'boolean'
+      default: true
+      order: 3
```

Comment:
This has the same order as the item above it. (Which throws all subsequent items off as well.)